Automattic/Gravatar-SDK-Android
# QuickEditor: Login Intro screen UI

Closes #494

### Description

This is a subtask of #485 that implements the new UI of the Intro login screen. I've removed the automatic OAuth flow start, but the proper logic will be implemented in the separate PR(subtask #495)

In one separate commit, I have renamed the `ErrorSection` to `CtaSection`. I believe this name better matches this UI component.

https://github.com/user-attachments/assets/47988f78-84da-4fe5-86f1-728e5930e93b



### Testing Steps
1. Start the demo app
2. Tap the Logout button
3. Tap the avatar to launch the QE
4. Confirm you can see the new updated UI
5. Tap Continue
6. Approve the authorization
7. Confirm you can see the loading
